config.yaml

```
@@ -36,3 +36,13 @@ options:
     default: true
     type: boolean
     description: Enable opensearch-knn
+
+  profile:
+   description: |
+      Profile representing the scope of deployment, and used to tune resource allocation.
+      Allowed values are: "production" or "default".
```

only `production`, `testing and `staging` are approved https://docs.google.com/document/d/1bq7KyUAPIIcUWGgifW05p40v7lGXs3PhMznM2ECaMKY/edit
```suggestion
      Allowed values are: "production" or "testing".
```